corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -413,6 +413,18 @@ hqDefine("app_manager/js/details/screen", function () {
             self.initColumnAsColumn(self.columns()[i]);
         }
 
+        self.caseTileRowMax = ko.computed(() => _.max([self.columns().length + 1, 7]));
```

Comment:
I think I'd like to have this `7` be a constant now that it's used in two different places, but I'm not sure where the right place to store it would be. Alternately, `column.js` could just initialize an empty array of options and this could be the source of truth for the default minimum, since it's updating immediately anyway.

Reply:
That makes sense to me.  You could add a comment over there to clarify if helpful.

I'm trying to think of some way to make that observable reference this observable directly, rather than doing the subscribe > update stuff manually, but I think it's too late in the day

Reply:
I looked at accessing `screen.caseTileRowMax` from the column model, but it seems like the column loads before the screen's observables are accessible, and the `caseTileRowMax` needs to load in the columns first to know how many there are, so this just seemed to save some headaches there.

Left it as `7` since it does need to be >= 1 (so _why not_ 7) and added a comment. https://github.com/dimagi/commcare-hq/pull/33667/commits/b688d566a8dfc28419c1a9a4278ad9ee122bc485